src/components/composite/layout/App/App.tsx

```diff
@@ -23,19 +22,21 @@ const App = ({ children }): React.ReactElement => {
 
   useUserScope(user);
 
-  const { data } = useServerMetadataQuery({
-    onCompleted: () => {
+  const { services } = useServerMetadata();
+
+  useEffect(() => {
+    if (services.length) {
       console.table({
         clientName: process.env.REACT_APP_NAME,
         clientVersion: process.env.REACT_APP_VERSION,
-        serverName: data?.metadata.services[0].name,
-        serverVersion: data?.metadata.services[0].version,
+        serverName: services[0].name,
```

Comment:
this is not safe; there could be multiple services and the order is not fixed

Reply:
We are going to have a much more granular service, already up to 3 back-end services, and this needs to be reviewed as an approach, whether that kind of table makes sense at all.